### ISSUE TITLE
Create /tables/<name> route

### ISSUE THREAD

**Issue by MartinAltmayer** [MartinAltmayer/pokerserver#8]
Return
```
players: {<position>: nameOrNone},
balances: {<playerName>: <balance>}
smallBlind: <amount>,
bigBlind: <amount>,
openCards: [....],
mainPot: <amount>,
sidePots: <???>,
bets: {<playerName>: <bet>},
currentPlayer: <name or None>,
smallBlindPlayer: <name>,
bigBlindPlayer: <name>,
dealer: <name>,
isClosed: <bool>,

# Following fields are private information. Only send if UUID is sent with request
cards: [...],
canJoin: <bool>
}
```

Cards are encoded like `Ad` or `10c` with the symbols `A, K, Q, J, 2, ..., 10` and the colors `h, s, d, c`.

**Comment by MartinAltmayer** [comment]
We need to restructure the tables relation so that it stores not only a list of players, but also the position. The classical way would be a separate 'players' relation with fields table-id, position, playername. Are you working on something like this in this issue, or shall I create a new issue? In any case this must be done before working on the action routes (join etc.).

**Comment by AndreasMalecki** [comment]
Okay. I will do that and add the fields "balance", "cards", and "bet" as well.

**Comment by AndreasMalecki** [comment]
I altered the output, as there is no additional information in which player is small or big blind. This can be deduced during the game if necessary. The player information now contains table_id, balance, cards, name, bet, and position.

**Comment by AndreasMalecki** [comment]
I still have to add some tests.
